### ISSUE TITLE
No information of the Current/Selected page on the selected pagination control is announced by the screen reader.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#1113]
**Describe the bug**
If the information of the current/selected page is not announced by the narrator, screen reader dependent user will not get the information of the selected page and they may get confused.

**To Reproduce**

1. Open code pen URL-[datagrid pagination (codepen.io)](https://codepen.io/seambr02/pen/KKreLbM)
2. Turn on narrator with 'Ctrl+ Win + Enter' key.
3. Now navigate to the Pagination controls present below the table.
4. Navigate to the Selected link one.
5. Observe the issue.

Alternatively link your repo with a sample project that can be run.
[datagrid pagination (codepen.io)](https://codepen.io/seambr02/pen/KKreLbM)

**Expected behavior**
When we navigate with tab key on the selected Pagination controls for e.g., 1, it should also announce the information that it is the current page, for example link one current page. This can be achieved by defining aria-current property for the controls.

**Desktop (please complete the following information):**
OS Build: Windows 11 Enterprise 22H2 (25267.1250)
Edge Dev Browser: Version 111.0.1652.0 (Official build) dev (64-bit)

**Attachments:**

Uploading MAS1.3.1_No information of  
![Code_Snip_No information of the current page on the selected pagination control is announced by the screen reader](https://github.com/radzenhq/radzen-blazor/assets/93735775/bdfb3e7d-88ef-41a1-9e62-3a49b0457e7d)

https://github.com/radzenhq/radzen-blazor/assets/93735775/ae29a38c-a7c5-445d-9408-3604770fef73

the current page pagination control is announced by the screen reader..mp4…

**Comment by akorchev** [comment]
> This can be achieved by defining aria-current property for the controls.

We would gladly accept a PR adding this feature.

**Comment by enchev** [comment]
https://github.com/radzenhq/radzen-blazor/pull/1205
